README.md

```diff
@@ -258,7 +253,7 @@ test('invoke onDragEnter when dragenter event occurs', async () => {
       )}
     </Dropzone>
   )
-  const { container } = render(ui)
+  const { container, rerender } = render(ui)
```

Comment:
in line 260 I think you have to change the arguments of flushPromises

Reply:
Thanks for reporting this. I've just updated it.